src/expr/src/vector_op/regexp.rs

```diff
@@ -142,7 +152,7 @@ fn regexp_match(text: &str, regex: &RegexpContext) -> Option<ListValue> {
     // If there are multiple captures, then the first one is the whole match, and should be
     // ignored in PostgreSQL's behavior.
     let skip_first = regex.regex.captures_len() > 1;
-    let capture = regex.regex.captures(text)?;
+    let capture = regex.regex.captures(text).unwrap()?;
```

Comment:
In which case does the `captures()` return an error?

Reply:
> Returns the capture groups for the first match in text. If no match is found, then Ok(None) is returned.

Thus I think it's safe to call `unwrap()` here 🤔️?

Reply:
> Returns the capture groups for the first match in text. If no match is found, then Ok(None) is returned.

But the document above doesn't mention the error. 🤔️

Reply:
Seems that the only `Err` will be generated from `captures` here is in the inner `fancy_regex::run`, which is a runtime error.
<img width="568" alt="CleanShot 2023-09-15 at 13 16 35@2x" src="https://github.com/risingwavelabs/risingwave/assets/91381303/39c6e2cf-5735-49d5-bf67-b35b1c3efb52">

Reply:
And according to this `Err`, I prefer to let it panic since some really strange errors are happening 🤔️?

Reply:
Okay!